.github/workflows/zero-touch-issues.yml

```
@@ -0,0 +1,170 @@
+            [[ "$NUMBER" =~ ^[0-9]+$ ]] || continue
+            ISSUE=$(gh issue view "$NUMBER" --repo "$REPO" --json number,title,body,labels,assignees,state)
+            [[ "$(jq -r '.state' <<<"$ISSUE")" == "OPEN" ]] || continue
+            LABELS=$(jq -r '[.labels[].name] | join("\n")' <<<"$ISSUE")
+            ASSIGNEES=$(jq -r '[.assignees[].login] | join("\n")' <<<"$ISSUE")
+            grep -qx 'copilot-swe-agent' <<<"$ASSIGNEES" && continue
+            grep -Eq '^(automation/hold|security|upstream/apple)$' <<<"$LABELS" && continue
+
+            TITLE=$(jq -r '.title' <<<"$ISSUE")
+            BODY=$(jq -r '.body // ""' <<<"$ISSUE")
+            TEXT="$TITLE
+$BODY"
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)